cmd/klaunch.go

```diff
@@ -15,17 +15,17 @@ Launch an experiment in Kubernetes.
 
 	$ iter8 k launch -c load-test-http --set url=https://httpbin.org/get
 
-To locally render the Kubernetes experiment, use the dry option.
+Use the dry option to simulate a Kubernetes experiment. This creates the manifest.yaml file.
```

Comment:
Perhaps add the phrase " but does not deploy it." to be more explicit?

Reply:
Fixed this documentation.